src/components/OSCALControlProse.js

```diff
@@ -169,7 +169,11 @@ function getTextSegment(text, key) {
   }
   return (
     <Typography component="span" key={key}>
-      <OSCALMarkupLine>{text}</OSCALMarkupLine>
+      {text.includes("\n") ? (
+        <OSCALMarkupMultiLine>{text}</OSCALMarkupMultiLine>
+      ) : (
+        <OSCALMarkupLine>{text}</OSCALMarkupLine>
+      )}
```

Comment:
OSCALMarkupLine will wrap everything and ignore "\n" because it is designed to only return a single line. That is why if the input contains "\n" we need to use OSCALMarkupMultiLine.

Reply:
I am thinking this is issues with branching... That change _should_ be in `develop` already. 

https://github.com/EasyDynamics/oscal-react-library/blob/3a96c641983ceef29cb004ff5c255c7f835ae44d/src/components/OSCALControlProse.js#L171-L177